js/src/attribute-mapping/attribute-mapping-category-control.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import AppSelectControl from '.~/components/app-select-control';
+import TreeSelectControl from '.~/components/tree-select-control';
+import useCategoryTree from '.~/hooks/useCategoryTree';
+
+const SELECT_TYPES = {
+	ALL: 'ALL',
+	EXCEPT: 'EXCEPT',
+	ONLY: 'ONLY',
+};
```

Comment:
By the way, since #1691 has been merged, and this PR will need to resolve conflicts, maybe these constants could be changed to shared ones when resolving conflicts:
- https://github.com/woocommerce/google-listings-and-ads/blob/05c7f0fcf1cd3d072300026edd3133cc99cd3f56/js/src/attribute-mapping/attribute-mapping-table-categories.js#L35
- https://github.com/woocommerce/google-listings-and-ads/blob/05c7f0fcf1cd3d072300026edd3133cc99cd3f56/js/src/attribute-mapping/attribute-mapping-table-categories.js#L61